Documentation/6.0.2/README.rst

```diff
@@ -0,0 +1,1515 @@
+    +--------------------------------------+------------+----------------------------------+-----------------+----------------------+-----------------+
+    | 75810b99-c372-463c-8684-f0d7b4e5743e | overcloud  | 1c60ab81cc924fe78355a76ee362386b | CREATE_COMPLETE | 2018-03-27T07:26:28Z | None            |
+    +--------------------------------------+------------+----------------------------------+-----------------+----------------------+-----------------+
+
+
+2. Run ``nova list`` to view the Overcloud Compute and Controller nodes.
+
+::
+
+    [stack@director ~]$ nova list
+    +--------------------------------------+------------------------+--------+------------+-------------+---------------------+
+    | ID                                   | Name                   | Status | Task State | Power State | Networks            |
+    +--------------------------------------+------------------------+--------+------------+-------------+---------------------+
+    | 437ff73b-3615-48cc-a9cf-ed0790953577 | overcloud-compute-0    | ACTIVE | -          | Running     | ctlplane=192.0.2.60 |
+    | 797e7a74-eb96-49fb-87e7-9e6955e70c70 | overcloud-compute-1    | ACTIVE | -          | Running     | ctlplane=192.0.2.58 |
+    | a7ef35db-4230-4fcd-9411-a6329f4747c9 | overcloud-compute-2    | ACTIVE | -          | Running     | ctlplane=192.0.2.59 |
+    | a0548879-0931-4b2c-bbe9-2733e4566d64 | overcloud-controller-0 | ACTIVE | -          | Running     | ctlplane=192.0.2.57 |
+    +--------------------------------------+------------------------+--------+------------+-------------+---------------------+
+
+
+3. Verify that the services are running.
+
+
+4. Check the VRS and VSC connection on an Overcloud Compute node.
+
+::
+
+    [heat-admin@overcloud-compute-1 ~]$ sudo ovs-vsctl show
+    cc87b725-7107-4917-b239-8dea497f5624
+        Bridge "alubr0"
+            Controller "ctrl1"
+                target: "tcp:101.0.0.21:6633"
+                role: master
+                is_connected: true
+            Controller "ctrl2"
+                target: "tcp:101.0.0.22:6633"
+                role: slave
+                is_connected: true
+            Port "alubr0"
+                Interface "alubr0"
+                    type: internal
+            Port svc-spat-tap
+                Interface svc-spat-tap
+                    type: internal
+            Port svc-pat-tap
+                Interface svc-pat-tap
+                    type: internal
+            Port "svc-rl-tap1"
+                Interface "svc-rl-tap1"
+            Port "svc-rl-tap2"
+                Interface "svc-rl-tap2"
+        ovs_version: "6.0.2-21-nuage"
+
+
+Phase 11 (Optional) For SR-IOV, manually install and run the topology collector
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Please see the Openstack Queen Nuage customer documentation for Topology collector under installation-and-configuration-topology-collection-agent-and-lldp section.
```

Comment:
Revise the text as follows:

See the "Installation and Configuration: Topology Collection Agent and LLDP" section in the *Nuage VSP OpenStack Neutron ML2 Driver Guide*.

Reply:
done